packages/blocks/plus_minus/src/index.js

```diff
@@ -0,0 +1,1048 @@
+   * given argument ID and mutates the callers to match.
+   * @param {string} argId The argId of the argument to remove.
+   * @this Blockly.Block
+   */
+  minus: function(argId) {
+    if (!this.argIds_.length) {
+      return;
+    }
+    this.removeArg_(argId);
+    Blockly.Procedures.mutateCallers(this);
+  },
+
+  /**
+   * Adds an argument to the block and updates the block's parallel tracking
+   * tracking arrays as appropriate.
+   * @param {string} opt_name An optional name for the argument.
+   * @param {string} opt_varId An optional variable ID for the argument.
+   * @this Blockly.Block
+   * @private
+   */
+  addArg_: function(opt_name, opt_varId) {
+    if (!this.arguments_.length) {
+      const withField = new Blockly.FieldLabel(
+          Blockly.Msg['PROCEDURES_BEFORE_PARAMS']);
+      this.getInput('TOP')
+          .appendField(withField, 'WITH');
+    }
+
+    const name = opt_name || Blockly.Variables.generateUniqueNameFromOptions(
+        Blockly.Procedures.DEFAULT_ARG, this.arguments_);
+    const variable = Blockly.Variables.getOrCreateVariablePackage(
+        this.workspace, opt_varId, name, '');
+    const argId = Blockly.utils.genUid();
+
+    this.addVarInput_(name, argId);
+    if (this.getInput('STACK')) {
+      this.moveInputBefore(argId, 'STACK');
+    } else {
+      this.moveInputBefore(argId, 'RETURN');
+    }
+
+    this.arguments_.push(name);
+    this.varIds_.push(variable.getId());
+    this.argumentVarModels_.push(variable);
+    this.argIds_.push(argId);
+  },
+
+  /**
+   * Removes the argument associated with the given argument ID from the block.
+   * @param {string} argId An ID used to track arguments on the block.
+   * @private
+   */
+  removeArg_: function(argId) {
+    if (!this.getInput(argId)) {
+      return;
+    }
+    // TODO: My life would be so much happier if this returned a boolean instead
+    //  of throwing an error. I'm saying at least 15% more joy.
+    this.removeInput(argId);
+    if (this.arguments_.length == 1) { // Becoming argumentless.
+      this.getInput('TOP').removeField('WITH');
+    }
+
+    const index = this.argIds_.indexOf(argId);
+    this.arguments_.splice(index, 1);
+    this.varIds_.splice(index, 1);
+    this.argumentVarModels_.splice(index, 1);
+    this.argIds_.splice(index, 1);
+  },
+
+  /**
+   * Appends the actual inputs and fields associated with an argument to the
+   * block.
+   * @param {string} name The name of the argument.
+   * @param {string} argId The UUID of the argument (different from var ID).
+   * @this Blockly.Block
+   * @private
+   */
+  addVarInput_: function(name, argId) {
+    const nameField = new Blockly.FieldTextInput(name, this.validator_);
+    nameField.onFinishEditing_ = this.finishEditing_.bind(nameField);
+    nameField.createdVarIds_ = [];
+
+    this.appendDummyInput(argId)
+        .setAlign(Blockly.ALIGN_RIGHT)
+        .appendField(new FieldMinus(argId))
+        .appendField('variable:') // Untranslated!
```

Comment:
You use Blockly.Msg for most of the field text in this PR.  Is this one untranslated because we don't have a message in Blockly that matches what you need?

Reply:
Yeah, there are [lots of strings](https://github.com/google/blockly/blob/07762ff4da3c713f7592c80052b1fa7cadb461a2/msg/messages.js#L143) that /contain/ the world variable, but none that just say 'variable' :/